docs/contributing/index.md

```diff
@@ -1,15 +1,67 @@
 # Contributing
 
-!!! warning
+This page explains how to contribute to Autoware.
 
-    Under Construction
+First, confirm the [Code of conduct](#contribution-workflows) section for the general manners in contributions.
+
+See the [Contribution workflows](#contribution-workflows) section for the workflows.  
+Also, confirm the following our guidelines depending on your situation:
+
+- [Coding guidelines](coding-guidelines/index.md): when you write source code
+- [Testing guidelines](testing-guidelines/index.md): when you write tests
+- [Documentation guidelines](documentation-guidelines/index.md): when you write documents
+- [Pull request guidelines](pull-request-guidelines/index.md): when you submit pull requests
+- [Discussion guidelines](discussion-guidelines/index.md): when you discuss with other contributors
+- [Support guidelines](../help/support-guidelines.md): when you need any help
+
+It is recommended to read the [Contributing pages in ROS 2 Docs](https://docs.ros.org/en/rolling/Contributing.html) as well because Autoware's guidelines generally depend on them.
+
+For general information about OSS contributions, [Open Source Guides](https://opensource.guide/) by GitHub is a good resource.
+
+!!!info
+
+    If you want to apply a new or different rule, you can open a new discussion thread and propose it.
+
+## Code of conduct
+
+We welcome contributions from everyone.  
+To ensure our community stays open and healthy, we adhere to the [Contributor Covenant](https://www.contributor-covenant.org/), a widely used [code of conduct](https://github.com/autowarefoundation/autoware/blob/main/CODE_OF_CONDUCT.md) adopted by many other communities listed [here](https://www.contributor-covenant.org/adopters/).
+
+Everyone participating in the Autoware community is expected to follow the code of conduct.  
+If someone in the community happens to be violating these terms, let the maintainers know, and we will address it as soon as possible.
+
+!!!warning
+
+    Note that contributions that ignore guidelines are not contributions but nuisances.
 
 ## Contribution workflows
 
-### Issues
+There are several types of contributions and the workflows differ between the types.
+See the description for your contribution type.
 
 ### Discussions
 
-### Small pull requests
+You can contribute to Autoware by facilitating discussions:
+
+- Find a bug and open a new issue.
+- Propose an idea of features that will enhance Autoware.
+- Join an existing discussion and express your opinion.
+- Organize discussion so far for others contributors.
```

Comment:
```suggestion
- Organize discussions to get opinions of other contributors.
```
I think you've meant this?

Reply:
@xmfcx What I wanted to say is:
- Sometimes people say only what you like and make others confused.
- In that case, we need some facilitators.

Reply:
I've added ` about their ideas.` does this work?

Reply:
Hmm, I personally prefer a simpler one.
Even `Organize discussions for other contributors.` looks good to me.

FYI: https://developers.google.com/style/translation?hl=en#write-short,-clear,-and-precise-sentences

Reply:
Sure, I've updated it.